test/specs.js

```diff
@@ -148,10 +148,19 @@ describe('module-alias', function () {
         expectAliasesToBeImported()
       })
 
-      it('should import settings from any config file', function () {
+      it('should import settings from any config file using absolute path', function () {
         moduleAlias(path.join(__dirname, 'src', 'default_module_alias', 'config.js'))
 
         expectAliasesToBeImported()
+
+        // Deleting config.js from require cache since we have to re-require it for next test.
+        delete require.cache[path.join(__dirname, 'src', 'default_module_alias', 'config.js')]
```

Comment:
I would rather do it in an `afterEach` in this `context`. The idea is to avoid depending on the order of the tests.

Reply:
Great catch, @Kehrlann. `config.js` is not being used by other (2) tests. Should I add it to `afterEach` anyway or move these 2 tests to a new `context`?

Reply:
Happy new year ! 🎉

I think just in the `afterEach` in this `context` is fine.

Reply:
A happy new year too! :smile: 

I moved it in the `afterEach` and pushed the latest changes.